modules/ROOT/pages/apply-configure-json-threat-task.adoc

```diff
@@ -9,30 +9,38 @@ endif::[]
 >s|Policy Name|JSON Threat Protection
 >s|Summary|Protects against malicious JSON in API requests
 >s|Category|Security
->s|Since Mule Version|3.8.0
+>s|Available from Mule version|3.8.0
```

Comment:
Do not use "from" in this context, to mean "starting with," because "from" has other similar meanings.
Replace with "Available in Product Name + x.x.x + and later"

Reply:
That is how it is in the file that I see. I don't know why it did not update it for you.